README.md

```diff
@@ -13,16 +13,18 @@ var cleankill = require('cleankill');
 To register a handler:
 
 ```js
-cleankill.onInterrupt(function(done) {
-  // do things.
-  done();
+cleankill.onInterrupt(() => {
```

Comment:
Expressions can be more than one line, arrow function expressions should be able to be too. But, like I said, nit.